test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -41,6 +41,9 @@ def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
 
 @pytest.mark.parametrize("ec2_instance_type", SMDEBUG_EC2_CPU_INSTANCE_TYPE, indirect=True)
 def test_smdebug_cpu(training, ec2_connection, region, cpu_only, py3_only):
+    # TODO: Remove this line and provide the required solution.
```

Comment:
Nit: can you update the comment to be `TODO: Remove this once test timeout has been debugged (failures especially on m4.16xlarge)`

Reply:
Updated